praw/__init__.py

```diff
@@ -1294,6 +1299,59 @@ def is_oauth_session(self):
         """Return True when the current session is an OAuth2 session."""
         return isinstance(self._authentication, set)
 
+    def leave_contributor(self, subreddit=None, fullname=None):
+        """Abdicate approved submitter status in a subreddit. Use with care.
+
+        Callable upon an instance of Subreddit with no arguments.
+        See :meth:`~praw.__init__.AuthenticatedReddit.leave_status` for
+        more details.
+
+        :returns: The json response from the server.
+        """
+        return self.leave_status(subreddit, fullname, status='contributor')
+
+    def leave_moderator(self, subreddit=None, fullname=None):
+        """Abdicate moderator status in a subreddit. Use with care.
+
+        Callable upon an instance of Subreddit with no arguments.
+        See :meth:`~praw.__init__.AuthenticatedReddit.leave_status` for
+        more details.
+
+        :returns: The json response from the server.
+        """
+        self.evict(self.config['my_mod_subreddits'])
+        return self.leave_status(subreddit, fullname, status='moderator')
+
+    @decorators.restrict_access(scope='modself', mod=False)
+    def leave_status(self, subreddit=None, fullname=None, status=None):
```

Comment:
Can we make this private? Or is there a reason you think it should be available?